src/command_modules/azure-cli-find/azure/cli/command_modules/find/_params.py

```diff
@@ -0,0 +1,16 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import CliArgumentType, register_cli_argument
+
+# pylint: disable=line-too-long
+
+query = CliArgumentType(
+    options_list=('--q', '-q'),
```

Comment:
I think I'm leaning toward `--search-query` / `-q`. Term has specific meaning in indexers and query parsers, so we may want to use that later. I feel like `-s` less intuitive than `-q`. Sound good, @derekbekoe?